boards/st/stm32n6570_dk/stm32n6570_dk_common.dtsi

```diff
@@ -48,6 +55,23 @@
 	};
 };
 
+&i2c2 {
+	clock-frequency = <I2C_BITRATE_FAST>;
+
+	gt911: gt911@d5 {
+		compatible = "goodix,gt911";
+		reg = <0x5d>;
+		reset-gpios = <&gpioe 4 GPIO_ACTIVE_LOW>;
+	};
+
+	clocks = <&rcc STM32_CLOCK(APB1, 22)>,
+		 <&rcc STM32_SRC_CKPER I2C2_SEL(1)>;
+
+	pinctrl-0 = <&i2c2_scl_pd14 &i2c2_sda_pd4>;
+	pinctrl-names = "default";
+	status = "okay";
```

Comment:
Prefer the usual layout: first the properties then the subnode?
```c
&i2c2 {
	status = "okay";
	pinctrl-0 = <&i2c2_scl_pd14 &i2c2_sda_pd4>;
	pinctrl-names = "default";
	clocks = <&rcc STM32_CLOCK(APB1, 22)>,
		 <&rcc STM32_SRC_CKPER I2C2_SEL(1)>;
	clock-frequency = <I2C_BITRATE_FAST>;

	gt911: gt911@5d {
		compatible = "goodix,gt911";
		reg = <0x5d>;
		reset-gpios = <&gpioe 4 GPIO_ACTIVE_LOW>;
	};
};
```

Reply:
Fixed